src/components/slider/slider.scss

```diff
@@ -137,6 +148,58 @@ md-slider *::after {
   border-color: md-color($md-accent);
 }
 
+.md-slider-thumb-label {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  position: absolute;
+  left: -($md-slider-thumb-label-height / 2);
+  top: $md-slider-thumb-label-top;
+  width: $md-slider-thumb-label-width;
+  height: $md-slider-thumb-label-height;
+  border-radius: max($md-slider-thumb-label-height, $md-slider-thumb-label-width);
+
+  transform: scale(0.4) translate3d(0, (-$md-slider-thumb-label-top + 10) / 0.4, 0);
+  transition: 300ms $swift-ease-in-out-timing-function;
```

Comment:
We should probably specify which properties we want to transition.  It may not be necessary now, but it can be really annoying to debug if a transition applies to an incompatible property.  As was my case with Tooltip a few weeks ago.